v2/tools/generator/internal/astmodel/package_import_set.go

```diff
@@ -55,6 +55,15 @@ func (set *PackageImportSet) AddImportsOfReferences(refs ...PackageReference) {
 	}
 }
 
+// AddImportsForPackageReferenceSet ensures this set includes an import of all the references in
+// the specified set.
+// Adding a reference already in the set is fine.
+func (set *PackageImportSet) AddImportsForPackageReferenceSet(refs *PackageReferenceSet) {
```

Comment:
minor: 
```suggestion
func (set *PackageImportSet) AddImportsOfPackageReferenceSet(refs *PackageReferenceSet) {
```

To match the naming of the above functions?

Reply:
It would have to be `AddImportsOfPackageReferencesFromSet` which is getting too long, even for me. I think there's room to improve these names, but didn't come up with a viable approach for this PR.